nfcreader/src/main/java/company/tap/nfcreader/internal/NFCApplication.java

```diff
@@ -0,0 +1,31 @@
+package company.tap.nfcreader.internal;
+
+import android.app.Application;
+import android.util.Log;
+
+import com.flurry.android.FlurryAgent;
+import com.flurry.android.FlurryPerformance;
+
+
+/**
+ * Created by AhlaamK on 4/5/20.
+ * <p>
+ * Copyright (c) 2020    Tap Payments.
+ * All rights reserved.
+ **/
+public class NFCApplication extends Application {
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        new FlurryAgent.Builder()
+                .withDataSaleOptOut(false) //CCPA - the default value is false
+                .withCaptureUncaughtExceptions(true)
+                .withIncludeBackgroundSessionsInMetrics(true)
+                .withLogLevel(Log.VERBOSE)
+                .withPerformanceMetrics(FlurryPerformance.ALL)
+                .build(this, "PTT9D949M57SP7WS45MJ");
```

Comment:
Please refer to the SRS 1.7.2 , any key should be saved in `.properties` file 
You can create config.properties file to save the values, and add reference to the value as a `buildConfigField`
See the [documentation](https://developer.android.com/studio/build/gradle-tips#remove-private-signing-information-from-your-project) for more details about using the property files and build config fields

Reply:
fixed